include/aie/Dialect/XLLVM/IR/XLLVMAIE2IntrOps.td

```diff
@@ -341,6 +341,39 @@ def VectorExtractElem32I512IntrOp :
 
 // ----- MAX ELEMENT -----
 
+def VectorMaxLt8IntrOp :
+    AIEVec2_IntrOp<"vmax.lt8",
+        [TypeIs<"res",
+            LLVM_StructOf<[
+                VectorOfLengthAndType<[64], [I8]>,
+                I32]>
```

Comment:
Since result[0] and operands have 64 lanes, I would expect result[1] to be 64 bits.

Reply:
Good catch. I will update both the min and max ops PRs.